requirements.txt

```diff
@@ -2,7 +2,7 @@
 -r requirements_production.txt
 
 # Requirements for development and tests in alphabetical order
-coverage
+coverage>=4.0a5,<4.1.0
```

Comment:
I would like not to define versions for development dependencies. In different then the productive dependencies, the versions are not tested, so the tools work or do not work irrespective of the file requirements.txt (for the productive dependencies the command 'openslides' checks the dependencies in setup.py which is the same as the versions in requirements_production.txt).

I think it's ok for a developer to call ```pip install -r requirements.txt -U```  from time to time. And there should be no harm, if a new version of development tool breaks our workprocess. At least we know, that a new version was released.